src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -565,6 +566,40 @@ public void requestTeamReviewers(List<GHTeam> teams) throws IOException {
                 .send();
     }
 
+    /**
+     * Set the base branch on the pull request
+     *
+     * @param newBaseBranch
+     *            the name of the new base branch
+     * @throws IOException
+     *             the io exception
+     */
+    public void setBaseBranch(String newBaseBranch) throws IOException {
+        owner.root.getClient()
+                .sendRequest(
+                        root.createRequest().method("PATCH").with("base", newBaseBranch).withUrlPath(getApiRoute()),
+                        null);
+    }
```

Comment:
This call returns the updated `GHPullRequest` in JSON. We should parse it and return it.